### PR TITLE
Clarify that the DOC is under CC BY-NC-SA 2.5

### PR DESCRIPTION
**Pull request by gpotter2**
- Clarify the doc vs code licenses.

closes https://github.com/secdev/scapy/issues/3870

### CODE REVIEW

**Comment by gpotter2** [comment]
@guedou @p-l- Hi. Could you confirm that this is fine?
Doc is under a Non-Commercial License, but that sounds... just fine to me.

**Comment by evverx** [comment]
FWIW packagers usually ship documentation separately and distros where it matters just don't ship the scapy documentation any more:
https://gitlab.com/redhat/centos-stream/rpms/scapy/-/commit/6239c6c2fcb09ba961f68b19f20c27d80cec2e7a
https://salsa.debian.org/pkg-security-team/scapy/-/commit/415bf9d78e38563551523ee9e87c1fb03e034a1d

so it shouldn't prevent the code from being distributed via classic packages at least.

**Comment by gpotter2** [comment]
Yeah. I take this opportunity to drop a slightly off-topic rant, aimed at whoever might come back to this PR in the future.

For a long time I was wondering if we should try to relicense the doc under not-NC license, pressured by package maintainers, but at the end of the day I'm still unsure what the big deal is. Nowadays 99% of users google "scapy documentation", so it's already debatable what impact doc packages really have.

It sounds reasonable to me to enforce that packages on a distro allow commercial use, and we are more than fine with Scapy being used in products or commercially (etc.), as you usually get good-quality contributions back from private companies that have both time and money, and interesting use cases. But enforcing documentation to also be commercial sounds a bit funky to me. I personally probably don't want companies to sell repackaged "guides" that just a copy/paste of the documentation, nor do I see a real usage for legitimate commercial use of documentation on the top of my mind.

I'm probably missing the reasons why this matters to so many distributions, but wording it a "DFSG infringement" and "do that else we'll remove the doc package, for the sake of freedom !" when we're only really talking about documentation sounds a bit like some sort of hallucinated delirium (feel free to correct me if I'm missing the point, as I've yet to see a convincing explanation).

**Review by p-l- — APPROVED** [review]

**Pull request merged**
